.github/workflows/scaffold-tfmigrate.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
           private_key: ${{ secrets.APP_PRIVATE_KEY }}
 
       - uses: suzuki-shunsuke/tfaction/scaffold-tfmigrate@main
-        id: scaffold-working-dir
+        id: scaffold-tfmigrate
```

Comment:
```suggestion
```

Thank you for your contribution!
You're right, but I think the id isn't used so we can simply remove the id.